AutocompleteClientTests/FW/Logic/Request/TrackPurchaseRequestBuilderTests.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TrackPurchaseRequestBuilderTests.swift
+//  AutocompleteClientTests
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import XCTest
+@testable import ConstructorAutocomplete
+
+class TrackPurchaseRequestBuilderTests: XCTestCase {
+
+    fileprivate let testACKey = "testKey123213"
+    fileprivate let customerIDs = ["custIDq3éû qd", "womp womp"]
+    fileprivate let sectionName = "some section name@"
+
+    fileprivate var encodedCustomerIDs: String = ""
+    fileprivate var encodedSectionName: String = ""
+
+    fileprivate var builder: RequestBuilder!
+
+    override func setUp() {
+        super.setUp()
+        self.encodedCustomerIDs = customerIDs.joined(separator: ",").addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)!
+        self.encodedSectionName = self.sectionName.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)!
+        self.builder = RequestBuilder(apiKey: testACKey)
+    }
+
+    func testTrackPurchaseBuilder() {
+        let tracker = CIOTrackPurchaseData(customerIDs: self.customerIDs)
+        builder.build(trackData: tracker)
+        let request = builder.getRequest()
+        let url = request.url!.absoluteString
+
+        XCTAssertEqual(request.httpMethod, "GET")
+        XCTAssertTrue(url.hasPrefix("https://ac.cnstrc.com/autocomplete/TERM_UNKNOWN/purchase?"))
+        XCTAssertTrue(url.contains("customer_ids=\(encodedCustomerIDs)"), "URL should contain the customer ID.")
```

Comment:
`customer_ids` should be `item_ids`, because customers use that as `id` to refer to any item. The name `customer_id` is an internal one that shouldn't be exposed to api consumers.

Reply:
Also instead of sending a comma-separated list, could we send multiple parameters, like `item_ids=id1&item_ids=id2&item_ids=id3...`? This is how we're currently accepting arrays in GET parameters in the API & it avoids problems with cases when the `customer_id` contains a comma for example.